src/main/java/com/hackathonorganizer/hackathonwriteservice/team/model/Team.java

```diff
@@ -0,0 +1,17 @@
+package com.hackathonorganizer.hackathonwriteservice.team.model;
+
+import javax.persistence.*;
+
+@Entity
+public class Team {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "hackathon_id")
+    private Long hackathonId;
+
+    @Column(name = "owner_id")
+    private Long ownerId;
+}
```

Comment:
jeszcze Set<Long> teamMemberIds by się przydało, najprościej pewnie będzie to zmapować z tabelą przez `@ElementCollection`